src/js/components/TextInput/doc.js

```diff
@@ -51,8 +51,8 @@ export default (TextInput) => {
       `Function that will be called when the user selects a suggestion.
 The suggestion contains the object chosen from the supplied suggestions.`
     ),
-    placeholder: PropTypes.string.description(
-      'Placeholder text to use when no value is provided.'
+    placeholder: PropTypes.node.description(
```

Comment:
should this be oneOfType ?

Reply:
It could be, although I believe technically a string is a node. Are you thinking more for documentation purposes?

Reply:
yup :)